baseline-profiles/app/build.gradle.kts

```diff
@@ -107,6 +111,7 @@ dependencies {
     implementation(libs.coil.kt.compose)
 
     implementation(libs.androidx.runtime.tracing)
+    implementation(libs.androidx.profileinstaller)
```

Comment:
should we have this in the `main` branch? This should be added with the macrobench / baseline profile modules

Reply:
Yeah you are right. Removed the dependency in [628a128](https://github.com/android/codelab-android-performance/pull/43/commits/628a12878bbd218bc3adc1e08213a0fa9e48c716)